tests/regression/test_pause_resume.py

```diff
@@ -161,38 +135,6 @@ def test_revert_second_stop_resume(self):
         with brownie.reverts("CONTRACT_IS_ACTIVE"):
             contracts.lido.resume({"from": contracts.voting})
 
-    @pytest.mark.skip(
-        reason="Second call of pause/resume staking is not reverted right now."
-        "It maybe should be fixed in the future to be consistent, "
-        "there's not a real problem with it."
-    )
-    def test_revert_second_pause_resume_staking(self):
-        contracts.lido.pauseStaking({"from": contracts.voting})
-
-        with brownie.reverts(""):
-            contracts.lido.pauseStaking({"from": contracts.voting})
-
-        contracts.lido.resumeStaking({"from": contracts.voting})
-
-        with brownie.reverts(""):
-            contracts.lido.resumeStaking({"from": contracts.voting})
-
-    def test_revert_second_pause_resume_staking_module(self, stranger):
-        contracts.staking_router.pauseStakingModule(1, {"from": contracts.deposit_security_module})
-
-        with brownie.reverts(encode_error("StakingModuleNotActive()")):
-            contracts.staking_router.pauseStakingModule(1, {"from": contracts.deposit_security_module})
-
-        contracts.staking_router.grantRole(
-            web3.keccak(text="STAKING_MODULE_RESUME_ROLE"),
-            stranger,
-            {"from": contracts.agent},
-        )
-        contracts.staking_router.resumeStakingModule(1, {"from": stranger})
-
-        with brownie.reverts(encode_error("StakingModuleNotPaused()")):
-            contracts.staking_router.resumeStakingModule(1, {"from": stranger})
```

Comment:
The `pauseStakingModule` and `resumeStakingModule` methods have been removed from the Staking Router contract.
The `setStakingModuleStatus`, which replaces those deprecated methods, is already covered by tests.
Therefore, I deleted this test to avoid duplication.